transient-aws/hive-example/cluster_bare_amis.conf

```diff
@@ -0,0 +1,194 @@
+
+#
+# Copyright (c) 2016 Cloudera, Inc. All rights reserved.
+#
+
+#
+# Simple AWS Cloudera Director configuration file with automatic role assignments
+# that works as expected if you use a single instance type for all cluster nodes
+#
+
+#
+# Cluster name
+#
+
+name: ETL-Transient
+
+#
+# Cloud provider configuration (credentials, region or zone and optional default image)
+#
+
+provider {
+    type: aws
+
+    accessKeyId: "REPLACE_ME"
+    secretAccessKey: "REPLACE_ME"
+
+    publishAccessKeys: false
+
+    region: us-west-2[REPLACE_ME]
+
+    subnetId: subnet-41b93525[REPLACE_ME]
```

Comment:
The value here should be removed as it won't be applicable for anyone outside cloudera-launchpad